app/models/comment.rb

```diff
@@ -25,7 +24,7 @@ class Comment < ApplicationRecord
   before_validation :evaluate_markdown, if: -> { body_markdown }
   validate :permissions, if: :commentable
   validates :body_markdown, presence: true, length: { in: BODY_MARKDOWN_SIZE_RANGE }
-  validates :body_markdown, uniqueness: { scope: BODY_MARKDOWN_UNIQUENESS_SCOPES }
+  validates :body_markdown, uniqueness: { scope: %i[user_id ancestry commentable_id commentable_type] }
```

Comment:
@rhymes Why did we remove the variable?

Reply:
Because rubocop uses a static checker and it's not going to detect the scoped variables and complain forever that we don't have a unique index on this row :D 

Rubocop woes

Reply:
Ah I see, thanks for the explanation!

Reply:
I'm not sure how I feel about this. Rubocop is a tool that is supposed to help us if we need to change reasonable code to please it, something feels off.